client/my-sites/comments/comment/comment-actions.jsx

```diff
@@ -64,12 +64,11 @@ export class CommentActions extends Component {
 			postId,
 			siteId,
 			unlike,
-			updateLastUndo,
 		} = this.props;
 
 		const alsoUnlike = commentIsLiked && 'approved' !== status;
 
-		updateLastUndo( null );
+		invoke( this.props, 'updateLastUndo', null );
```

Comment:
Why does [invoke](https://lodash.com/docs/4.17.4#invoke) work vs calling `updateLastUndo` directly?

Reply:
Could we not pass though the function in main?
```
{ canModerateComments && <Comment commentId={ commentId } refreshCommentData={ true } /> }
```

Reply:
The thing is, there is no need of `updateLastUndo` in comment view at all.

We can default it to `noop`, we can check for undefined (e.g. `!! updateLastUndo && updateLastUndo()`), or we can let `invoke` take care of it.
I don't have a strong opinion here.
Maybe `noop` is more clear/more used in Calypso, as opposed to `invoke`.